setup.py

```diff
@@ -32,6 +32,7 @@
     "defusedxml~=0.7.1",
     "pytz~=2021.1",
     "requests~=2.25",
+    "clean-text~=0.6.0"
```

Comment:
Last nitpick comment, Shak. Like the rest of the lists on this file, can we keep the comma `,` here? i.e. `"clean-text~=0.6.0",`

Reply:
Done!